.devops/package.nix

```diff
@@ -0,0 +1,119 @@
+{ lib
+, config
+, stdenv
+, cmake
+, ninja
+, pkg-config
+, git
+, python3
+, openmpi
```

Comment:
Llama.cpp seems to use implementation-agnostic apis, so we don't have to hardcode openmpi (there's a top-level `mpi` attribute which the users are free to reset to e.g. mpich instead of openmpi): https://github.com/ggerganov/llama.cpp/blob/708e179e8562c2604240df95a2241dea17fd808b/CMakeLists.txt#L324-L326